src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,73 @@
+package duke;
+
+import parser.Parser;
+import storage.Storage;
+import tasklist.TaskList;
+import ui.Ui;
+
+/**
+ * Initializes the Java Duke Program, loads the task list
+ * from the hard disk and gets the UI running.
+ *
+ * @author (Sruthi)
+ */
+public class Duke {
+    private final Ui ui;
+    private final Storage storage;
+    private final TaskList taskList;
```

Comment:
Since taskList is a collection, would it be a good idea to use a plural name like tasks instead?